app/views/partials/_menu.html.haml

```diff
@@ -0,0 +1,13 @@
+%h1 Today's Menu
+- @categories.each do |category|
+    %h2= category.category
+    %div{id: dom_id(category)}
+    - category.products.each do |product|
+        %h4= product.name
+        %p= product.description
+        %p= product.price
+        - if user_signed_in?  
+            - if current_user.owner
+                = link_to "Edit #{category.category}", edit_product_path(product.id)
```

Comment:
Does this mean that you only have one product per category? I still don't get the logic here...

Reply:
We have updated this now, so now we hav 3 products per category and each product has a unique edit product link.